src/Select.js

```diff
@@ -583,11 +583,12 @@ export default class Select extends Component<Props, State> {
     if (!this.state.isFocused) {
       this.openAfterFocus = true;
       this.focusInput();
-    } else if (!this.state.menuIsOpen) {
```

Comment:
This looked like a legit bug that the upgrade uncovered.

Reply:
it is indeed, thanks for this!